Docs-update (#1829)
mui/mui-toolpad: docs/data/toolpad/deployment.md

# Deployment

> <p class="description">Deploying a Toolpad app is a 3-step process and quite like any other react application.</p>

Reviewer: ```suggestion
<p class="description">Deploying a Toolpad app is a 3-step process and quite like any other React application.</p>
```

Reviewer 2: ```suggestion
<p class="description">Deploying a Toolpad app is like deploying any other React application and has 3 steps:</p>
```

To serve your app locally:

```sh
npm start
```

> This command will create an optimized production build for the Toolpad app and will output what files it has generated and how large each file is:

Reviewer: ```suggestion
This command will create an optimized production build for the Toolpad app and will output the generated files:
```

```sh
npm run build
```

> Once the build has been made. You can deploy it to any service of your choice!

Reviewer: ```suggestion
Once the build has been made, you can deploy it to any service of your choice!
```
